spec/test_app/app/controllers/basechurch/v1/posts_controller_spec.rb

```diff
@@ -65,10 +65,10 @@
     context 'with an authenticated user' do
       before do
         request.headers['Content-Type'] = 'application/vnd.api+json'
-        request.headers['X-User-Email'] = logged_user.email
-        request.headers['X-User-Token'] = logged_user.session_api_key.access_token
+        request.headers['X-User-Email'] = user.email
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.